frosh/code-quality-meta/0.2/root/rector.php

```diff
@@ -39,6 +39,7 @@
         SymfonySetList::ANNOTATIONS_TO_ATTRIBUTES,
         LevelSetList::UP_TO_PHP_81,
         ShopwareSetList::SHOPWARE_6_5_0,
+        ShopwareSetList::SHOPWARE_6_6_0,
```

Comment:
that set does not exist yet and I think it won't make sense to have both at same time 🤔

Reply:
I have found the class here: https://github.com/FriendsOfShopware/shopware-rector/tree/main/config/v6.6. 
In my local setup, this class is already available, so I thought it wasalready released